terraform/development/per_review_app/main.tf

```diff
@@ -0,0 +1,114 @@
+  cloudfront_header_name                            = data.terraform_remote_state.development.outputs.front_door_cloudfront_header_name
+  cloudfront_header_password                        = data.terraform_remote_state.development.outputs.front_door_cloudfront_header_password
+  database_name                                     = "${local.prefix}-${local.default_database_name}"
+  database_partial_connection_string_parameter_name = data.terraform_remote_state.development.outputs.database_rds_partial_connection_string_parameter_name
+  ecs_deployment_role_name                          = data.terraform_remote_state.development.outputs.application_roles_ecs_deployment_role_name
+  ecs_security_group_id                             = data.terraform_remote_state.development.outputs.application_security_group_ecs_security_group_id
+  ecs_task_execution_role_arn                       = data.terraform_remote_state.development.outputs.application_roles_ecs_task_execution_role_arn
+  ecs_task_execution_role_id                        = data.terraform_remote_state.development.outputs.application_roles_ecs_task_execution_role_id
+  ecs_task_role_arn                                 = data.terraform_remote_state.development.outputs.application_roles_ecs_task_role_arn
+  export_bucket_details                             = data.terraform_remote_state.development.outputs.cds_export_details
+  govuk_notify_api_key_secret_arn                   = data.terraform_remote_state.development.outputs.application_secrets_govuk_notify_api_key_secret_arn
+  load_balancer_arn_suffix                          = data.terraform_remote_state.development.outputs.front_door_load_balancer_arn_suffix
+  load_balancer_listener_arn                        = data.terraform_remote_state.development.outputs.front_door_load_balancer_listener_arn
+  os_data_key_secret_arn                            = data.terraform_remote_state.development.outputs.application_secrets_os_data_key_secret_arn
+  private_subnet_ids                                = data.terraform_remote_state.development.outputs.networking_private_subnet_ids
+  rails_env                                         = local.rails_env
+  rails_master_key_secret_arn                       = data.terraform_remote_state.development.outputs.application_secrets_rails_master_key_secret_arn
+  redis_connection_string                           = module.redis.redis_connection_string
+  sentry_dsn_secret_arn                             = data.terraform_remote_state.development.outputs.application_secrets_sentry_dsn_secret_arn
+  sns_topic_arn                                     = data.terraform_remote_state.development.outputs.monitoring_sns_topic_arn
+  vpc_id                                            = data.terraform_remote_state.development.outputs.networking_vpc_id
+}
+
+module "redis" {
+  source = "../../modules/elasticache"
+
+  snapshot_retention_limit = 5

```

Comment:
Changed this to `0` (removed redis backups) and added a checkov ignore comment in the `per_review_app` `main.tf` entrypoint. The comment had to go here rather than inside the elasticache module, to avoid it falsely affecting the other envs

Reply:
Turns out this doesn't work in the way I thought, it will still ignore it for all envs, so I've had to move it into the elasticache module now 😃